spec/spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+require 'chefspec'
+require 'chefspec/berkshelf'
+
+ChefSpec::Coverage.start! { add_filter 'crowd' }
+
+CENTOS_7 = {
+  platform: 'centos',
+  version: '7.2.1511',
+}.freeze
+
+CENTOS_6 = {
+  platform: 'centos',
+  version: '6.8',
+}.freeze
+
+ALL_PLATFORMS = [
+  CENTOS_6,
+  CENTOS_7,
+].freeze
+
+RSpec.configure do |config|
+  config.log_level = :warn
```

Comment:
Not sure about this, but based on our `code_generator` cookbook I think this should be `:fatal`
```suggestion
  config.log_level = :fatal
```